database/seeds/DatabaseSeeder.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use App\Models\Application;
+use App\Models\Build;
+use App\Models\Device;
+use Illuminate\Database\Seeder;
+
+class DatabaseSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        factory(Application::class, 20)
+			->create()
+			->each(function (Application $application) {
+				factory(Build::class, 10)
+					->create(['application_id' => $application->id])
+					->each(function (Build $build) use ($application) {
+						factory(Device::class)
+							->create([
+								'build_id' => $build->id,
+								'application_id' => $application->id,
+							]);
+					});
+			});
+    }
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common processes. Try to leave a link to the process described so that the reader knows how to resolve the chore.  -->
**💣 chore:** I think there is something wrong with the code formatting 🤔 ‏